src/App.jsx

```diff
@@ -19,6 +20,7 @@ function App() {
             <Header />
             <Nav />
             <Routes>
+                <Route path="/*" element={<Error message="Page not found!" />} />
```

Comment:
This route would usually be placed last, since it's a dynamic catch all kind of path, it'd want to be placed last since it acts as a fall-back. You wouldn't want to run the risk of this path being found before another one by accident